recipes/openssl/1.x.x/conanfile.py

```diff
@@ -683,16 +683,20 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "OpenSSL"
         self.cpp_info.names["cmake_find_package_multi"] = "OpenSSL"
         if self._use_nmake:
+            libsuffix = "d" if self.settings.build_type == "Debug" else ""
             if self._full_version < "1.1.0":
-                self.cpp_info.libs = ["ssleay32", "libeay32"]
+                self.cpp_info.components["SSL"].libs = ["ssleay32" + libsuffix]
+                self.cpp_info.components["Crypto"].libs = ["libeay32" + libsuffix]
```

Comment:
```suggestion
                self.cpp_info.components["ssl"].libs = ["ssleay32" + libsuffix]
                self.cpp_info.components["crypto"].libs = ["libeay32" + libsuffix]
```
I would keep the names of these components lower case and set its names.
```
self.cpp_info.components["crypto"].names["cmake_find_package"] = "Crypto"
self.cpp_info.components["ssl"].names["cmake_find_package"] = "SSL"
```
Also, don't forget to set the `pkg_config` name: openssl generated 3 `.pc` files by default:
`libcrypto.pc`, `libssl.pc` and `openssl.pc`

Reply:
Done